### ISSUE TITLE
Community style: indent ns args two spaces

### ISSUE THREAD

**Issue by pesterhazy** [kkinnear/zprint#13]
When formatting clojure.spec.test, I get the following diff

```
 (ns clojure.spec.test
-  (:refer-clojure :exclude [test])
-  (:require
-   [clojure.pprint :as pp]
-   [clojure.spec :as s]
-   [clojure.spec.gen :as gen]
-   [clojure.string :as str]))
+ (:refer-clojure :exclude [test])
+ (:require [clojure.pprint :as pp]
+           [clojure.spec :as s]
+           [clojure.spec.gen :as gen]
+           [clojure.string :as str]))
```

This looks strange because the second line is indented by one space. (By the way, I'm not talking about the fact that `[clojure.pprint :as pp]` is moved to the first line, which is a change I like.)

Without specifying community style I get the correct

```
 (ns clojure.spec.test
   (:refer-clojure :exclude [test])
-  (:require
-   [clojure.pprint :as pp]
-   [clojure.spec :as s]
-   [clojure.spec.gen :as gen]
-   [clojure.string :as str]))
+  (:require [clojure.pprint :as pp]
+            [clojure.spec :as s]
+            [clojure.spec.gen :as gen]
+            [clojure.string :as str]))
```

I think that the first argument of `ns` should be indented by two, not one spaces, even when choosing community style.

- Assuming that by "community style" you refer to @bbatsov's style guide, his style guide uses two space: https://github.com/bbatsov/clojure-style-guide#comprehensive-ns-declaration
- Same for the clojure repository (as seen above)
- It's true that the guide has the imfamous "one-space-indent" rule: https://github.com/bbatsov/clojure-style-guide#one-space-indent but my interpretation would be that `ns` is an implicit exception to this rule.

Is there a way to specify `ns` as an exception and to include this exception in `:style :community`?

**Comment by pesterhazy** [comment]
To reproduce in a clojure git repo:

```
boot -d boot-fmt/boot-fmt:0.1.4 fmt -m diff -f src/clj/clojure/spec/test.clj | head -40
```

or

```
boot -d boot-fmt/boot-fmt:0.1.4 fmt -m diff -f src/clj/clojure/spec/test.clj -o '{:style :community}' | head -40
```

**Comment by pesterhazy** [comment]
I guess my point is `ns` should be indented like `do`?

**Comment by pesterhazy** [comment]
Actually I meant `when`. I went ahead and created a PR:

https://github.com/kkinnear/zprint/pull/14

**Comment by pesterhazy** [comment]
Great, I've released a new version of boot-fmt based on 0.2.12.

I tried it with a couple of open-source projects and zprint using community style does a marvellous job of keeping modifications of well-edited Clojure code to a minimum.

Also thanks for linking to boot-fmt on the homepage!
